src/samtools/samtools_index/config.vsh.yaml

```diff
@@ -0,0 +1,74 @@
+name: samtools_index
+namespace: samtools
+description: Index SAM/BAM/CRAM files.
+keywords: [index, bam, sam, cram]
+links:
+  homepage: https://www.htslib.org/
+  documentation: https://www.htslib.org/doc/samtools-index.html
+  repository: https://github.com/samtools/samtools
+references: 
+  doi: [10.1093/bioinformatics/btp352, 10.1093/gigascience/giab008]
+license: MIT/Expat
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --input
+        type: file
+        description: Input file name
+        required: true
+        must_exist: true
+  - name: Outputs
+    arguments:
+      - name: --output
+        alternatives: -o
+        type: file
+        description: Output file name
+        required: true
+        must_exist: false
+        example: out.bam.bai
+  - name: Options
+    arguments:
+      - name: --bai
+        alternatives: -b
+        type: boolean_true
+        description: Generate BAM index
+      - name: --csi
+        alternatives: -c
+        type: boolean_true
+        description: |
+          Create a CSI index for BAM files instead of the traditional BAI 
+          index. This will be required for genomes with larger chromosome 
+          sizes.
+      - name: --min_shift
+        alternatives: -m
+        type: integer
+        description: |
+          Create a CSI index, with a minimum interval size of 2^INT.
+      - name: --multiple
+        alternatives: -M
+        type: boolean_true
+        description: |
+          Interpret all filename arguments as alignment files to
+          be indexed individually. (Without --multiple, filename arguments
+          are interpreted solely as per the second synopsis.)
```

Comment:
```suggestion
```

If all file arguments are not set to `multiple: true`, this argument shouldn't be added.